PSKoans/Koans/Constructs and Patterns/AboutRegularExpressions.Koans.ps1

```diff
@@ -565,9 +565,11 @@ Describe 'Meditative Examples' {
             '01234'
             '+14255556789'
         )
+        
+        $sanitizedNumbers = $phoneNumbers -replace '____' 
```

Comment:
This will either need to be in a `BeforeAll { }` block or we'll need to figure out a better way to put this one everywhere it's needed. This will work OK in pester 4.x but it will probably be problematic or break outright in 5.x.

One possible option is to define a hashtable in a `BeforeAll { }` and add things into that so you can reference them from multiple `It` blocks.

Reply:
[`BeforeAll` docs](https://pester.dev/docs/commands/beforeall/)

Reply:
@vexx32  thanks for the feedback, let me know if this is what you were thinking in the latest commit

Reply:
@udubnate thanks! This will work. 🤔 Not sure about having the blank itself in the `BeforeAll` though, it'll probably be mostly confusing for folks since it's a ways apart from the koan itself

What if we reframed that koan and put `$sanitizedNumbers` in as a literal declaration into the `BeforeAll`, so we can make that koan assertion something like:

```ps1
$phoneNumbers -replace '____' | Should -Be $sanitizedNumbers`
```

And `$sanitizedNumbers` can just use the array that's currently attached to the `Should -Be` in this koan as its definition in the `BeforeAll`. Does that make sense?

Reply:
yeah makes sense, I'll take a look today or the next few days. ill ping you if I have more questions

Reply:
should meet your requirement now, let me know